src/util/db/dbconnection.cpp

```diff
@@ -250,7 +250,9 @@ bool initDatabase(QSqlDatabase database, StringCollator* pCollator) {
     // v.data() returns a pointer to the handle
     sqlite3* handle = *static_cast<sqlite3**>(v.data());
     // Ensure initialization. database.open() might not do it
-    sqlite3_initialize();
+    int rc = sqlite3_initialize();
+    if( rc ) qDebug() << "sqlite3_initialize failed with the code: " << rc;
```

Comment:
Please check for `!= SQLITE_OK` and log a warning instead of only a debug message.

Code formatting is not correct.

Reply:
Ok, I just copy&pasted what sqlite.c does. I will add that to be more explicit.
And.. what do you mean with code formatting? add { }  ?

Reply:
I have noticed that it is not a good idea to place this code between obtaining and checking the handle. Please move it either before or after those lines that belong together.

We don't expect a failure here, so please copy the VERIFY_OR_DEBUG_ASSERT style from the  `handle` check including the usage of `kLogger